.ruby-version

```diff
@@ -1 +1 @@
-2.7.7
\ No newline at end of file
+2.7.8
```

Comment:
I'm not sure if this is actually needed, my machine was throwing up audit errors with trivy, and the bump in Ruby patch version seemed to help.

The same audit run in CI doesn't show audit failures 🤷🏾‍♂️